feature/experimental/isis/ate_tests/isis_interface_hello_padding_enable_test/README.md

```diff
@@ -0,0 +1,99 @@
+# RT-2.6: IS-IS Hello-Padding  enabled at interface level
+
+## Summary
+
+*   Base IS-IS functionality and adjacency establishment.
+*   Verifies isis adjacency by changing MTU.
+
+## Procedure
+
+*   Configure IS-IS for ATE port-1 and DUT port-1.
+*   Configure DUT with global hello-padding enabled.
+*   Ensure that adjacencies are established with:
+    *   Interface level hello padding is enabled.
+    *   Verify that IPv4 and IPv6 IS-ISIS adjacency comes up fine.
+    *   Verify the output of ST path displaying the status of ISIS hello padding.
+    *   If we change the MTU on either side, then adjacency should not come up.
+    *   Verify that IPv4 and IPv6 prefixes that are advertised by ATE correctly installed into DUTs route and forwarding table.
+    *   TODO-Verify the Hellos are sent with Padding during adjacency turn-up if the padding is enabled adaptively/sometimes.
```

Comment:
Why is this a todo?

Reply:
There is no OC path available to check the above. Hence marked as TODO.